### PR TITLE
Environment provisioning

### PR DESCRIPTION
**Pull request by lenadoc**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/cloud/dev/spryker-cloud-commerce-os/environment-provisioning.md [hunk 1]
```diff
@@ -82,27 +107,53 @@ We can share the required credentials mentioned in the preceding documentation o
...
+
+### Site to Site VPN
+A Site-to-Site VPN (Virtual Private Network) is a type of network connection that enables secure communication between two or more geographically separated networks. This type of VPN allows two or more sites to establish a secure and encrypted connection over the internet or other public networks, creating a virtual private network between the two sites.
+- customer gateway IP address
+- IP ranges on the customer side that would need access to Spryker VPC
+Optional:
+- Device name on customer gateway
+- Custom BGP(Border Gateway Protocol) ASN
+- Dynamic BGP routes
+- Other Parameters of tunnels
```
Comment (lenadoc) [inline docs/cloud/dev/spryker-cloud-commerce-os/environment-provisioning.md:146]: @michalwroblewski-spryker what do list items lines 140 through 146 mean?

Comment (michalwroblewski-spryker) [reply docs/cloud/dev/spryker-cloud-commerce-os/environment-provisioning.md:146]: Those are configuration parameters that are required (or optional) to setup Site to Site VPN tunnel between our Spryker Cloud environment and Customer on-premise

File: docs/cloud/dev/spryker-cloud-commerce-os/environment-provisioning.md [hunk 2]
```diff
@@ -82,27 +107,53 @@ We can share the required credentials mentioned in the preceding documentation o
...
+
+In case IPsec is\will be needed, provide your internal subnet CIDR, so our Spryker VPC doesn’t overlap here. It is very important to evaluate this option during provisioning as Spryker can’t change this later once the environment is provisioned and in case it overlaps with customer CIDR in future there is no option that Spryker can provide except customers using different ranges.
+
+### Default Network Settings
+Each Spryker Cloud Commerce environment is using a dedicated VPC (Virtual Private Cloud) 
+The default CIDR used for:
+- first non-production environment is 10.105.0.0/16.
+- first production environment is 10.106.0.0/16.
```
[the remark this answers is not in the document]
Comment (michalwroblewski-spryker) [reply docs/cloud/dev/spryker-cloud-commerce-os/environment-provisioning.md:154]: This is default IP range (CIDR - https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) that is used to provision first non-production (at this moment Dev or Staging) environment.

File: docs/cloud/dev/spryker-cloud-commerce-os/environment-provisioning.md [hunk 3]
```diff
@@ -82,27 +107,53 @@ We can share the required credentials mentioned in the preceding documentation o
...
+A Site-to-Site VPN (Virtual Private Network) is a type of network connection that enables secure communication between two or more geographically separated networks. This type of VPN allows two or more sites to establish a secure and encrypted connection over the internet or other public networks, creating a virtual private network between the two sites.
+- customer gateway IP address
+- IP ranges on the customer side that would need access to Spryker VPC
+Optional:
+- Device name on customer gateway
+- Custom BGP(Border Gateway Protocol) ASN
+- Dynamic BGP routes
+- Other Parameters of tunnels
+
+In case IPsec is\will be needed, provide your internal subnet CIDR, so our Spryker VPC doesn’t overlap here. It is very important to evaluate this option during provisioning as Spryker can’t change this later once the environment is provisioned and in case it overlaps with customer CIDR in future there is no option that Spryker can provide except customers using different ranges.
```
[the remark this answers is not in the document]
Comment (michalwroblewski-spryker) [reply docs/cloud/dev/spryker-cloud-commerce-os/environment-provisioning.md:148]: Indeed this does not sound right ;) Let me rephrase it
